bedrock/firefox/templates/firefox/sync.html

```diff
@@ -94,9 +94,30 @@
         </div>
 
         <div class="show-fx-31-signed-out">
+        {% if not version %}
           <button class="sync-button" data-interaction="button click" data-element-action="Sync CTA"  id="cta-sync">
             {{ _('Get started with Sync') }}
           </button>
+        {% else %}
+          <form id="fxa-email-form" action="{{ url('firefox.accounts') }}" method="get">
+            <input type="hidden" id="variation" name="v" value="{{ version }}">
```

Comment:
I guess as long as we don't have any other tests running on `firefox/accounts` this will be fine, but we would need to be careful not to cross pollute tests while this test is running?